### ISSUE TITLE
unknown-metas reports the new 'also' meta as unknown

### ISSUE THREAD

**Issue by VasilevNStas** [objectionary/lints#1380]
## Problem

`unknown-metas` reports `+also` as an unknown meta, although the modern parser emits it (see `add-default-package.xsl` in eo-parser).

## Root cause

`src/main/resources/org/eolang/lints/metas/unknown-metas.xsl:15` whitelists a fixed set of meta heads (`package`, `architect`, `version`, `home`, `spdx`, `rt`, `lint`, `tests`, `unlint`, `import`, `alias`, `compile`, `repo`, `register`, `system`, ...) — `also` is not among them, so any `+also ...` is flagged as unknown.

## Minimal example

```eo
+package x
+architect yegor@256.com
+version 0.0.1
+home https://github.com/objectionary
+spdx SPDX-FileCopyrightText: Copyright (c) 2026 Objectionary.com
+spdx SPDX-License-Identifier: MIT
+also app

[] > main
```

`+also app` is reported as an unknown meta.

## Expected behavior

`also` should be added to the list of known metas (and kept in sync with the metas the parser can emit).

## Related

- eo-parser `add-default-package.xsl` (introduces `+also`).
- Existing packs under `packs/single/unknown-metas/`.

**Comment by 0crat** [comment]
@vasilevnstas Thanks for reporting a new bug! You've earned +12 points. By reporting bugs, you help our project improve its quality. If you find anything else in the repository that [doesn't look](https://www.yegor256.com/2018/02/06/where-to-find-more-bugs.html) as good as you might expect, [do not hesitate](https://www.yegor256.com/2014/04/13/bugs-are-welcome.html) to [report](https://www.yegor256.com/2018/04/24/right-way-to-report-bugs.html) it. Your running score is [+782](https://www.eolang.org/zerocracy/objectionary-vitals.html); don't forget to check your [Zerocracy account](https://www.zerocracy.com/) too).
